test/fixtable.test.ts

```diff
@@ -22,10 +22,11 @@ import { fixTables } from '../src/';
 const cw100 = td({ colwidth: [100] }, p('x'));
 const cw200 = td({ colwidth: [200] }, p('x'));
 
-function fix(table: Node) {
-  const state = EditorState.create({ doc: doc(table) });
+function fix(node: Node) {
+  const isdoc = node.type.name == 'doc';
+  const state = EditorState.create({ doc: isdoc ? node : doc(node) });
   const tr = fixTables(state);
-  return tr && tr.doc.firstChild;
+  return tr && (isdoc ? tr.doc : tr.doc.firstChild);
 }
```

Comment:
Could you explain the change to the `fix` function?

Reply:
Test functions like `eq` expect valid nodes for comparison. The expected behavior from `fixTables` for this case is to delete a node, so for this scenario the comparison should take place at the level of the parent node, i.e. the document.  To accomplish this, the `fix` function must be able to work directly with document nodes, not just the tables themselves.